HHS/OPRE-OPS
# feat: adds CAN Funding YTD table

## What changed

- adds CAN Funding YTD table
- UX Review fixes

## Issue

#2673 #3081 

## How to test

1. e2e tests pass
2. goto `/cans/504/funding`
3. select fy 2021
4. see table and cards light up

## Screenshots

<img width="994" alt="image" src="https://github.com/user-attachments/assets/c4a2f5f5-7ca9-47d4-b3cf-c7be451f5fb7">

## Definition of Done Checklist
